bundle/generated/embed.go

```diff
@@ -0,0 +1,6 @@
+package generated
+
+import _ "embed"
+
+//go:embed jsonschema.json
+var BundleSchema []byte
```

Comment:
Hard to tell significance from "generated", please move under `bundle/schema`.

Reply:
Also, please add a few unit tests to confirm that the contents is what we expect it to be.

I know the end-to-end tests ensure the schema itself is good. I would like to also confirm that the documentation strings are present for a few select paths. E.g. if we traverse the `resources.Job` type, can we find a non-empty description for the `name` field?